js/build.js

```diff
@@ -137,6 +137,10 @@ Fliplet.Widget.instance('chat', function (data) {
     multipleNameColumns = true;
   }
 
+  if (data.dataSourceId === 'none') {
+    delete data.dataSourceId;
```

Comment:
Will an undefined `data.dataSourceId` result in a visible error message to the user?

Reply:
Yes

Reply:
@tonytlwu actually no, it's console only. If you send me a copy I can display the relevant toast message.

Reply:
_Cannot connect to contact list. Please check it's configured correctly._

Reply:
Added and tested, @tonytlwu please review again.

Reply:
Is there any need in continuing with any of the subsequent code? I assume the component is useless without a contact list data source configured so it might as well just stop there.

Reply:
@tonytlwu it's better to continue because the console then spits out the required errors given the component is not configured. Same for the UI, which can behave accordingly. The same principle applies to other components in the system, e.g. form.